packages/react-app/helpers/Web3Context.js

```diff
@@ -283,6 +284,61 @@ export function Web3Provider({ children, network = "localhost", DEBUG = false, N
           />
         </div>
       );
+    } else {
+      networkDisplay = (
+        <div style={{ zIndex: 10, position: "absolute", right: 0, top: 80, padding: 16 }}>
```

Comment:
Please use the Chakra alert dialog component instead: you can incorporate this here: https://github.com/moonshotcollective/drecruit/blob/staging/packages/react-app/pages/index.js#L48

Reply:
@QEDK I'm changing this to use the AlertDialog, but I think that it's not want we want. The AlertDialog is a modal, blocking the access to the site. We want just tell the user that the network selected is wrong but allow the user to see the site and the profiles. Please, let me know what you think.

Reply:
@damianmarti This is intentional, but if you think it should be closable, it is trivial to implement it using `<AlertDialogCloseButton />` however, it should be a blocking option which means that the user **has** to interact with the modal to make it close (so no implementation to automatically close should be added).

Reply:
@QEDK I understand from my chats with @Cali93 that we want that the site can be viewed even if the user is not in the right network. If we want this, I think that we shouldn't use a modal alert dialog for this. I think we should use just a Chakra alert.

Reply:
@damianmarti Once the alert dialog is dismissed, the UI will be accessible again (using local provider). Essentially for someone who does not want to switch networks the flow is same except there's no annoying alert left over, if they attempt to connect to the wrong network again, the alert dialog triggers again, so it's consistent and there's no UI remnant, that's the intent here.

Reply:
Ok! Thanks!

Reply:
@QEDK I made this change and fixed the chainChanged callback (the network provider was not changed before). I tested it changing from the right network to a wrong one, back and forth, and it seem to be working fine now. The action "Request Contact Information" is shown or hidden when you go from the right network to a wrong one. Please review it and let me know if something is wrong.